vendor/init.go

```diff
@@ -66,22 +83,65 @@ var files = []file{
 	},
 }
 
+func addChrome() (err error) {
+	var chromeFile file
+	chromeFile.name = chromeFilename
+	chromeFile.browser = true
+	ctx := context.Background()
+	client, err := storage.NewClient(ctx, option.WithHTTPClient(http.DefaultClient))
+	if err != nil {
+		return fmt.Errorf("Cannot create a storage client for downloading the chrome browser: %s", err)
+	}
+	BktHandle := client.Bucket(storageBktName)
```

Comment:
Local variables should begin with lower-case letters.

Also, just call this "bkt" or "bucket", which is simpler.

https://github.com/golang/go/wiki/CodeReviewComments#variable-names